MultiTarget/MultiTargetIdentifiers.props

```diff
@@ -43,7 +43,8 @@
 
     <IsUwp Condition="'$(IsUwp)' == '' AND $(UwpTargetFramework.Contains('$(TargetFramework);')) AND '$(UwpTargetFramework)' != '' AND '$(MultiTargetsUwp)' == 'true'">true</IsUwp>
     <IsWinAppSdk Condition="'$(IsWinAppSdk)' == '' AND $(WinAppSdkTargetFramework.Contains('$(TargetFramework);')) AND '$(WinAppSdkTargetFramework)' != '' AND '$(MultiTargetsWasdk)' == 'true'">true</IsWinAppSdk>
-    <IsWindowsTfm Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'">true</IsWindowsTfm>
+    <TargetPlatformIdentifier>$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)'))</TargetPlatformIdentifier>
```

Comment:
Don't name it `TargetPlatformIdentifier`, better not to mess with well known properties set by the .NET SDK. Just name it something like `MultiTargetPlatformIdentifier` or whatever.

Reply:
Good catch, fixed in https://github.com/CommunityToolkit/Tooling-Windows-Submodule/pull/235/commits/c0d535d20494566b135b8f527e98d0b7b918ecbc